presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -10,16 +10,102 @@ it does not cause running queries to fail; instead new queries become queued.
 A resource group may have sub-groups or may accept queries, but may not do both.
 
 The resource groups and associated selection rules are configured by a manager, which is pluggable.
-Add an ``etc/resource-groups.properties`` file with the following contents to enable
-the built-in manager that reads a JSON config file:
+
+There are two built-in managers: file-based and database-based. Add an ``etc/resource-groups.properties``
+file with the appropriate configurations to enable one of the two managers.
+
+File Resource Group Manager
+---------------------------
+
+This manager reads the resource group configuration from a JSON config file.
 
 .. code-block:: none
 
     resource-groups.configuration-manager=file
     resource-groups.config-file=etc/resource-groups.json
 
-Change the value of ``resource-groups.config-file`` to point to a JSON config file,
-which can be an absolute path, or a path relative to the Presto data directory.
+* ``resource-groups.config-file``: a JSON file storing the configurations, which can be an absolute
+  path, or a path relative to the Presto data directory.
+
+Db Resource Group Manager
+-------------------------
+
+This manager periodically loads the resource group configuration from a database.
+
+.. code-block:: none
+
+    resource-groups.configuration-manager=db
+    resource-groups.config-db-url=jdbc:mysql://localhost:3306/config_db?user=presto_admin
+    resource-groups.max-refresh-interval=1m
+    resource-groups.exact-match-selector-enabled=true
+
+* ``resource-groups.config-db-url`` (required): Database URL to load configuration from.
+* ``resource-groups.max-refresh-interval``: Time period for which the cluster will continue to accept
+  queries after refresh failures cause configuration to become stale. The default is ``1h``.
+* ``resource-groups.exact-match-selector-enabled``: Boolean flag to enable usage of a selector requiring
+  an exact match of environment, source and query type.
+
+The configuration is loaded every second from the following tables under the specified db url. If the
```

Comment:
The paragraph and following table needs to be broken up and reordered

* First startup and automatic table creation 
* Document each table maybe with example row of data (including additional table for exact-match)
* Document how we suggest users add data to the table (maybe example insert statement, update statement), also should they do that from Presto via catalog, or some other SQL UI directly on the db (probably that)
* How it works (refresh interval...)